packages/pool/test/StateUtils.sol.js

```diff
@@ -606,7 +599,7 @@ describe("setAprUpdateStep", function () {
 describe("setProposalVotingPowerThreshold", function () {
   context("Caller is primary DAO Agent", function () {
     context(
-      "Proposal voting power threshold to be set is smaller than or equal to 10,000,000",
+      "Proposal voting power threshold to be set is smaller than or equal to 100%",
```

Comment:
We have an inconsistency here and in the Line 576, if 10,000,000 is 100%, then 500,000 is 5%, not 0,5%

Reply:
So we actually need to make `newAprUpdateStep = ONE_PERCENT.mul(5);`, but I think that is not actually that importatnt

Reply:
P.S. the same goes for line 77, but I think this number is actually 10%, so we just need to fix names for context(from line 309)

Reply:
Previously 1,000,000 was 1% and 100,000,000 was 100%
https://github.com/api3dao/api3-dao/blob/develop/packages/pool/contracts/StateUtils.sol#L72-L73
so the initial PR got the strings wrong, I fixed them.

Reply:
10,000,000 is actually 10%, I just forgot that the maximum proposal voting power threshold is 10% and not 100%. Pushed the fix.